src/common/types/vector.cpp

```diff
@@ -571,6 +571,10 @@ void Vector::Deserialize(idx_t count, Deserializer &source) {
 	}
 }
 
+nullmask_t Vector::getNullMask() {
```

Comment:
Also here, remove it please. How to get the nullmask of a vector depends on the type of vector, this is why the nullmask method is inside the vector specializations.